test/functional/autocmd/cmdline_spec.lua

```diff
@@ -92,8 +92,8 @@ describe('cmdline autocommands', function()
     -- cmdline was still executed
     feed('<cr>')
     screen:expect([[
-                                                                              |
-      ^lorem ipsum                                                             |
+      ^                                                                        |
+      {1:~                                                                       }|
```

Comment:
why isn't "lorem ipsum" displayed now?

Reply:
I don't know, I thought `semsg` and `msg_printf_attr` just printed a messages, so not sure why there's a functional difference in behavior.

Reply:
OK, I think the difference is that `semsg` cancels current command while `msg_printf_attr` doesn't.